### PR TITLE
Prevent scrolling on initial challenge run

### LINKED ISSUE #9256: Scroll to bottom of #scroll-locker
#### Challenge Name
https://www.freecodecamp.com/challenges/comparisons-with-the-logical-or-operator

#### Issue Description
This applies to any challenge with large contents.
The scroll to the bottom of #scroll-locker has some side effect, is scrolling the user on inital load.
We should not scroll when the JS does initial challenge run to populate tests

### PR DESCRIPTION
**Pull request by heldersepu**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/git-rebase#squashing-multiple-commits-into-one) them into one commit).
- [x] All new and existing tests pass the command `npm run test-challenges`. Use `git commit --amend` to amend any fixes.
#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)
#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue: Closes #9256 
#### Description

Adding an additional parameter to function displayTestResults that determines if we should scroll

### CODE REVIEW

**Comment by erictleung** [comment]
@heldersepu thanks for the PR. Can you squash your both your commits into one? We ask that each PR contain only one commit. Thanks!

**Comment by Bouncey** [comment]
I don't have time right now to pull this down and test.

Can be a big UX problem

cc @FreeCodeCamp/issue-moderators

**Comment by Bouncey** [comment]
I'm still getting the scroll animation on initial load of the challenge. I got it on `Exect Change` & `Inventory Update`

Would it help to tie the animation event to `submitBtn$` instead of `displayTestResults`?

**Comment by heldersepu** [comment]
@Bouncey I'm short on time too...
If we want a quick fix we should just comment the line that does the scrolling: 
https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/client/commonFramework/display-test-results.js#L24


I'm not sure why you still getting the animation the function `displayTestResults` is called from 3 places in the [end.js](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/client/commonFramework/end.js) only one is used on initial challenge run to populate tests:
https://github.com/heldersepu/FreeCodeCamp/blob/fix/Scroll-to-bottom-of-%23scroll-locker/client/commonFramework/end.js#L124

The call to `displayTestResults ` on initial load should be `common.displayTestResults(tests, false);`
Can you please debug end.js on your environment see why it still scrolling ?

File: client/commonFramework/end.js [hunk 1]
```diff
@@ -164,7 +164,7 @@ $(document).ready(function() {
             return common.updateOutputDisplay('' + err);
           }
           common.codeStorage.updateStorage(challengeName, originalCode);
-          common.displayTestResults(tests);
+          common.displayTestResults(tests, true);
```
Comment (Bouncey) [inline client/commonFramework/end.js]: changing this to `common.displayTestResults(tests, false);` seems to fix the issue.

You had only disabled the scroll on load for HTML challenges.

**Comment by Bouncey** [comment]
## **LGTM** 

Thanks for contributing :tada: :sparkles: :boom: :tada:

**Pull request merged**